packages/typescript-estree/src/parser.ts

```diff
@@ -417,3 +413,5 @@ export function parseAndGenerateServices<
 
 export { AST_NODE_TYPES } from './ast-node-types';
 export { ParserOptions };
+export { ParserServices };
+export { es };
```

Comment:
i'm unsure what name we should use when we are exporting all of `typedefs`

Reply:
@bradzacher you are expecting that exported name will be `TSESTree` in #120

should i change it here?

Reply:
i updated it already 👍